### PR TITLE
fix(ext/body): .bodyUsed for static body

### PR DESCRIPTION
**Pull request by marcosc90**
Fixes an edge case for `.bodyUsed` for static bodies when `.body` getter is called after consuming the body.

```javascript
const response = new Response("body");
await response.text();
console.log(response.bodyUsed); // true
response.body;
console.log(response.bodyUsed); // false
```

### CODE REVIEW

File: cli/tests/unit/response_test.ts [hunk 1]
```diff
@@ -90,3 +90,14 @@ Deno.test(function customInspectFunction() {
   );
   assertStringIncludes(Deno.inspect(Response.prototype), "Response");
 });
+
+Deno.test(async function responseBodyUsed() {
```
Comment (marcosc90) [inline cli/tests/unit/response_test.ts:94]: This fails on `main`

File: ext/fetch/22_body.js [hunk 2]
```diff
@@ -92,6 +94,8 @@
         if (consumed) {
           this.streamOrStatic = new ReadableStream();
           this.streamOrStatic.getReader();
+          this.streamOrStatic[_disturbed] = true;
```
Comment (lucacasonato) [inline ext/fetch/22_body.js:97]: Can you add a `readableStreamDisturb` function to `06_streams.js` instead so we can keep this symbol private to that module?

Comment (marcosc90) [reply ext/fetch/22_body.js]: done!

**Review by lucacasonato — APPROVED** [review]
LGTM, thanks!

**Pull request merged**